localstack-core/localstack/dev/run/__main__.py

```diff
@@ -237,6 +237,8 @@ def run(
         if pro is None:
             if os.getcwd().endswith("localstack-ext"):
                 pro = True
+            elif "localstack-pro-" in os.getcwd().split(os.pathsep)[-1]:
```

Comment:
Probably stupid question but is this correct for azure? I thought that was a different image so is that functionality implemented by `localstack.dev.run`?